runhouse/servers/http/http_utils.py

```diff
@@ -0,0 +1,69 @@
+import codecs
+import logging
+import re
+import sys
+from typing import List, Union, Optional
+
+from pydantic import BaseModel
+from ray import cloudpickle as pickle
+
+
+logger = logging.getLogger(__name__)
+
+
+class Message(BaseModel):
+    data: str
+
+
+class Response(BaseModel):
+    data: Union[None, str, List[str]]
+    error: Optional[str]
+    traceback: Optional[str]
+    output_type: str
+
+
+class OutputType:
+    EXCEPTION = "exception"
+    STDOUT = "stdout"
+    STDERR = "stderr"
+    SUCCESS = "success"  # No output
+    RESULT = "result"
+    RESULT_LIST = "result_list"
+
+
+def pickle_b64(picklable):
```

Comment:
why do we need to encode and pickle? is this for serializing python objects in text-based format?

Reply:
Yes, exactly. Pickle doesn't go over the wire nicely.